packages/styled-docs/pages/theme.mdx

```diff
@@ -1 +1,217 @@
+import { Box, Flex, Grid, Stack, useTheme } from "@trendmicro/react-styled-core";
+import {
+  ColorPalette,
+  ColorPalettes,
+  ColorWrapper,
+} from "../components/ColorPalette";
+import ThemeParser from "../components/ThemeParser";
+
 # Theme
+
+The theme object is where you define your application's color palette, type
+scale, font stacks, breakpoints, border radius values, and more. Theming is based on the
+[Styled System Theme Specification](https://styled-system.com/theme-specification)
+
+## Colors
+
+Add a `theme.colors` object to provide colors for your project. By default these
+colors can be referenced by the `color`, `borderColor`, and `backgroundColor`,
+`fill`, `stroke`, styles.
+
+We recommend adding palette that go from `10` - `100`.
+
+<ThemeParser theme="colors" />
+
+### Orange
+
+<ColorWrapper>
+  <ColorPalettes color="orange" />
+</ColorWrapper>
+
+### Yellow
+
+<ColorWrapper>
+  <ColorPalettes color="yellow" />
+</ColorWrapper>
+
+### Red
+
+<ColorWrapper>
+  <ColorPalettes color="red" />
+</ColorWrapper>
+
+### Magenta
+
+<ColorWrapper>
+  <ColorPalettes color="magenta" />
+</ColorWrapper>
+
+### Purple
+
+<ColorWrapper>
+  <ColorPalettes color="purple" />
+</ColorWrapper>
+
+### Blue
+
+<ColorWrapper>
+  <ColorPalettes color="blue" />
+</ColorWrapper>
+
+### Green
+
+<ColorWrapper>
+  <ColorPalettes color="green" />
+</ColorWrapper>
+
+### Teal
+
+<ColorWrapper>
+  <ColorPalettes color="teal" />
+</ColorWrapper>
+
+### Cyan
+
+<ColorWrapper>
+  <ColorPalettes color="cyan" />
+</ColorWrapper>
+
+### Gray
+
+<ColorWrapper>
+  <ColorPalettes color="gray" />
+</ColorWrapper>
+
+### BlackAlpha
+
+<ColorWrapper>
+  <ColorPalettes color="black" />
+</ColorWrapper>
+
+### WhiteAlpha
+
+<ColorWrapper>
+  <ColorPalettes color="white" />
+</ColorWrapper>
+
+
+## Typography
+
+To manage Typography options, the theme object supports the following keys:
+
+- `fonts` (font families)
+- `fontSizes`
+- `fontWeights`
+- `lineHeights`
+
+<ThemeParser theme="fonts" />
+<ThemeParser theme="fontSizes" />
+<ThemeParser theme="fontWeights" />
+<ThemeParser theme="lineHeights" />
+
+## Breakpoints
+
+To configure the default breakpoints used in responsive array values, add a
+breakpoints array to your theme. These values will be used to generate
+mobile-first (i.e. min-width) media queries, which can then be used to apply
+responsive styles.
+
+> For example, you can write `<Box fontSize={["14px", "16px"]}/>` to make the
+> fontSize will be responsive.
```

Comment:
Revise the sentence as below:

For example, you can write `<Box fontSize={["14px", "16px"]} />` to make the font size responsive.